draft-ietf-quic-recovery.md

```diff
@@ -1042,16 +1107,17 @@ UpdateRtt(latest_rtt, ack_delay):
   // Limit ack_delay by max_ack_delay
   ack_delay = min(ack_delay, max_ack_delay)
   // Adjust for ack delay if it's plausible.
+  adjusted_rtt = latest_rtt
   if (latest_rtt - min_rtt > ack_delay):
-    latest_rtt -= ack_delay
+    adjusted_rtt = latest_rtt - ack_delay
   // Based on {{?RFC6298}}.
   if (smoothed_rtt == 0):
-    smoothed_rtt = latest_rtt
+    smoothed_rtt = adjusted_rtt
```

Comment:
I'd suggest moving the adjusted_rtt calculation into the else and using latest_rtt here?  If not, I'd ensure smoothed_rtt and rttvar are set as a function of the same field, not one based on adjusted_rtt and the other latest_rtt.

Reply:
*facepalm* thanks for catching this. I believe this was a bug in the original pseudo-code and the ack_delay adjustment should have been in the else to begin with.